publish.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+#
+# publish.rb
+#
+# This script publishes the ev3dev website to the gh-pages branch of your
+# personal fork. This allows you to share a preview of your changes with others.
+#
+# Usage: ./publish.rb <gh-user-name>
+
+require 'tmpdir'
+
+if ARGV.count != 1
+    STDERR.puts "Usage: ./publish.rb <gh-user-name>"
+    exit(1)
+end
+
+Dir.mktmpdir do |tmp|
+    FileUtils.cp_r "_site/.", tmp
+    Dir.chdir tmp
+    system "git init"
+    git_url = "git@github.com:#{ARGV[0]}/ev3dev.github.io.git"
+    system "git remote add origin #{git_url}"
```

Comment:
What if I already have a remote configured that points to my fork, or already have something named `origin` configured? And this should use `https` instead so that it works on Windows.